src/components/tabs/tab.tsx

```diff
@@ -22,8 +23,10 @@ const Tab = (props: TabProps) => {
     </div >
   );
 };
-Tab.defaultProps = {
-  resourceData: {},
-  active: false,
-};
+Tab.defaultProps = Object.create({}, {
```

Comment:
Why do you need to use `Object.create` here?

Reply:
as default props works the props generated here are equal between components, for example, if I want a prop `number` to generate a random number if the user did not passed it, the random number number will be the same for every component occurrence, when you wrap default props with `Object.Create`, it will generate new props every time you render a component, the use case here is that **if the user did not pass any id, the tabs will generate random ones**

Reply:
Ok, Can you just create a test case to check this behavior?

Reply:
added the test case